src/tests/programs/io.rs

```diff
@@ -50,12 +50,77 @@ pub fn secretin_rast_10() -> syn::ItemFn {
     })
 }
 
+#[allow(dead_code)]
```

Comment:
Is this annotation really necessary on a public function? I think that since it is public, the linter cannot possibly know whether the code is dead, and hence doesn't generate a warning. (However, I might be missing some context.)

Reply:
Good point. Don't know.